go.mod

```diff
@@ -1,12 +1,56 @@
 module github.com/prometheus-community/prom-label-proxy
 
-go 1.13
+go 1.18
```

Comment:
let's upgrade it to go 1.17 first instead.

Reply:
just to clarify. Put go 1.17 in go.mod, but we still want to build this project with go 1.18.
Like that it will say that it is compatible with go 1.17

Reply:
Done.